launcher/ui/MainWindow.cpp

```diff
@@ -334,6 +337,14 @@ class MainWindow::Ui
         actionSettings->setShortcut(QKeySequence::Preferences);
         all_actions.append(&actionSettings);
 
+        actionUndoTrashInstance = TranslatedAction(MainWindow);
+        connect(actionUndoTrashInstance, SIGNAL(triggered(bool)), MainWindow, SLOT(undoTrashInstance()));
+        actionUndoTrashInstance->setObjectName(QStringLiteral("actionUndoTrashInstance"));
+        actionUndoTrashInstance.setTextId(QT_TRANSLATE_NOOP("MainWindow", "&Undo last instance deletion"));
```

Comment:
```suggestion
        actionUndoTrashInstance.setTextId(QT_TRANSLATE_NOOP("MainWindow", "&Undo Last Instance Deletion"));
```

for consistency with everything else in the menu bar

Also this action is always enabled, even when nothing is available to undo, maybe it could have a similar fix to the other undo action?

Reply:
Yes I am going to fix it after exams

Reply:
Great, good luck on exams!

Reply:
Thanks :)